std_msgs/Header header

uint16 gnss_week
float64 gnss_time

float64 heading
float64 pitch
float64 roll

float64 gyroscope_x
float64 gyroscope_y
float64 gyroscope_z

float64 acceleration_x
float64 acceleration_y
float64 acceleration_z

float64 latitude
float64 longitude
float64 altitude

float64 velocity_e
float64 velocity_n
float64 velocity_u

float64 velocity

uint16 nsv1
uint16 nsv2

char[2] status

uint8 age

uint8 warming